src/reducers/selectionReducer.js

```diff
@@ -35,7 +36,7 @@ const initialState = {
 };
 
 export default (state=initialState, { id: cardId, type }) => {
-  if(!cardId) {
+  if(!_.includes([SELECT, HOVER_IN, HOVER_OUT], type)) {
```

Comment:
Maybe using `Object.keys(stateTransitions)` would be better. It would make it future proof.